cc_plugin_imos/srs.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
             'sst_dtime'
         ]
 
-        self.time_units = r"^seconds since 1981-01-01.*$"
+        self.time_units = r"^seconds since 1981-01-01( 00:00:00.*|).*$"
```

Comment:
Might have a few too many ".*" in there?
This would match e.g. "seconds since 1981-01-01 23:59:59" -- unlikely perhaps, but you could make your pattern more specific if you think it's worth it
```suggestion
        self.time_units = r"^seconds since 1981-01-01( 00:00:00)?$"
```

Reply:
> Might have a few too many ".*" in there?
absolutely!! thanks